src/admin.rs

```diff
@@ -65,3 +88,41 @@ impl From<NativeFluvioAdmin> for FluvioAdmin {
         }
     }
 }
+
+macro_rules! impl_stream {
```

Comment:
you fancy. We should figure out how to use this for the `PartitionConsumerStream`